GAE/src/org/akvo/flow/api/app/WebFormServlet.java

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright (C) 2020 Stichting Akvo (Akvo Foundation)
+ *
+ *  This file is part of Akvo FLOW.
+ *
+ *  Akvo FLOW is free software: you can redistribute it and modify it under the terms of
+ *  the GNU Affero General Public License (AGPL) as published by the Free Software Foundation,
+ *  either version 3 of the License or any later version.
+ *
+ *  Akvo FLOW is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ *  without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ *  See the GNU Affero General Public License included below for more details.
+ *
+ *  The full license text can also be seen at <http://www.gnu.org/licenses/agpl.html>.
+ */
+
+package org.akvo.flow.api.app;
+
+import com.gallatinsystems.framework.rest.AbstractRestApiServlet;
+import com.gallatinsystems.framework.rest.RestRequest;
+import com.gallatinsystems.framework.rest.RestResponse;
+import com.google.appengine.api.utils.SystemProperty;
+
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.logging.Logger;
+
+
+/**
+ * Redirect to tech-consultancy url or show the error
+ */
+@SuppressWarnings("serial")
+public class WebFormServlet extends AbstractRestApiServlet {
+    private static final Logger log = Logger.getLogger(WebFormServlet.class.getName());
+
+    public WebFormServlet() {
+        setMode(JSON_MODE);
+    }
+
+    @Override
+    protected RestRequest convertRequest() throws Exception {
+        HttpServletRequest req = getRequest();
+        log.warning(req.getPathInfo());
+        RestRequest restRequest = new WebFormRequest();
+        restRequest.populateFromHttpRequest(req);
+        return restRequest;
+    }
+
+    @Override
+    protected RestResponse handleRequest(RestRequest req) throws Exception {
+        RestResponse res = new RestResponse();
+        res.setCode(String.valueOf(HttpServletResponse.SC_OK));
+        String surveyId = ((WebFormRequest)req).getSurveyId();
+        String url = String.format("https://tech-consultancy.akvotest.org/akvo-flow-web/%s/%s", SystemProperty.applicationId.get(), surveyId);
```

Comment:
@muloem how can we read the flow env `akvotest` value in flow app? so we can use `prod` depending deployment environment

Reply:
You would need to add it to the appengine-web config and feed it in through the `EnvServlet.java`

Reply:
https://github.com/akvo/akvo-flow-server-config/pull/600

Reply:
https://github.com/akvo/akvo-flow/pull/3583/commits/2ef5a43c4c6154763a78398ce1e729b34699bb1e